datasources/postresql/users_db/users_db.sql

```diff
@@ -5,8 +5,7 @@ CREATE TABLE Users (
     id          SERIAL NOT NULL PRIMARY KEY,
     email       VARCHAR(255) NOT NULL UNIQUE,
     username    VARCHAR(50) NOT NULL UNIQUE,
-    first_name  VARCHAR(50),
-    last_name   VARCHAR(50),
+    last_name   VARCHAR(255),
```

Comment:
Shouldn't last name also be removed here?

Reply:
You're right, I didn't remember to refactor the name to display_name 😂